data/common.yaml

```diff
@@ -61,8 +61,18 @@ scriptworker_users:
       worker_type_prefix: "tb-"
       worker_id_suffix: "tb"
       cot_product: "thunderbird"
+      # TODO: confirm if we need widevine for tb
       widevine_filename: "widevine_dep.crt"
       keychain_filename: "dep-signing.keychain"
+    vpnbld:
```

Comment:
moved dep building to dep worker now.
Haven't tested yet, will have to quarantine a dep worker to test it out.

Reply:
We're much more resource-constrained on the dep pools, so it may be safer to puppetize a prod worker while forcing it to use dep configs.